pom.xml

```diff
@@ -112,6 +114,9 @@
         <module>presto-testing-server-launcher</module>
         <module>presto-plugin-toolkit</module>
         <module>presto-resource-group-managers</module>
+        <module>presto-thrift-interface</module>
+        <module>presto-thrift-node</module>
```

Comment:
Originally, I wanted to add more tables to thrift node, so it won't be just tpch. However, I think it'll be more clear if I rename the project to presto-thrift-tpch-node.